packages/devtools_app/lib/src/shared/config_specific/logger/logger_io.dart

```diff
@@ -2,17 +2,24 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'package:logging/logging.dart';
+
 import 'logger.dart';
 
+final _log = Logger('main_log');
+
 void log(Object message, [LogLevel level = LogLevel.debug]) {
   switch (level) {
     case LogLevel.debug:
       print(message);
+      _log.info(message);
```

Comment:
just verifying - do we need to log and print everywhere?

Reply:
We should instead be registering a listener on `_log.onRecord` which handles the printing. See the example in the [package documentation](https://pub.dev/documentation/logging/latest/).